Widgets/Widgets/VarmintWidgetButton.cs

```diff
@@ -13,6 +13,7 @@ public class VarmintWidgetButton : VarmintWidgetLabel
         public float LineWidth { get; set; }
         public bool HasBorder { get; set; }
         public string BackgroundImage { get; set; }
+        public Vector2 TextOffset { get; set; }
```

Comment:
The textoffset here doesn't help the user very much because they have to manually maintain/update the offset every time the control dimensions change.  What we want is for the control to do as much as possible automatically .   The label control already handles text formatting in this way by using the content alignment. There is no need for a text offset if we set the content property to an image with a label.   Let's talk about this on Friday.